backend/FwLite/MiniLcm/Models/Entry.cs

```diff
@@ -2,6 +2,8 @@ namespace MiniLcm.Models;
 
 public record Entry : IObjectWithId<Entry>
 {
+    public static readonly Entry Empty = new();
```

Comment:
I don't love this because you then need to write code which detects if you have an empty entry and handle that, for example `CreateEntry(Entry.Empty)` should throw. If this does stick around then it should be changed to return a new object because `Entry` is mutable. So an empty entry could accidentally be modified and now everyone gets a non empty entry.